### ISSUE TITLE
论文表述上的疑问

### ISSUE THREAD

**Issue by zhangmin10001** [lnbxldn/Bridge#2]
如论文3.2中所述“Upon obtaining the final fusion mask, we integrate this refined depth supervision into a two-stage
training process. Initially, the D2I model undergoes pre-training on a large-scale dataset, primarily
using the teacher-generated pseudo-labels. Following this, a fine-tuning stage is performed. In this
fine-tuning stage, for regions covered by the fusion mask, we directly utilize the corresponding highprecision ground truth depth (Dgt) from the original synthetic data as training labels. This strategy
ensures the model first learns broad geometric consistency from vast pseudo-labeled data and then
refines its accuracy and detail with the most accurate supervision in high-precision, geometrically
consistent areas, thereby significantly improving depth estimation accuracy and detail.
” 其中在微调阶段，“we directly utilize the corresponding highprecision ground truth depth (Dgt) from the original synthetic data as training labels”对于D2I模型的训练，这里用的标签为什么是Dgt，我觉得应该是原始合成的RGB图Iorig，是我理解上哪里有问题吗?

**Comment by lnbxldn** [comment]
We apologize if the wording in our paper caused any misunderstanding. To clarify, Section 3.2 describes the construction of the MDE model's training set. In the first stage, the MDE model is trained on RGB images generated by D2I and pseudo-labels generated by a teacher model. Subsequently, it is fine-tuned leveraging the similarity mask of Dgt and Igen. We will revise this description in the paper for better clarity. Thank you for your valuable question!
